chapters/generative-models.md

```diff
@@ -355,23 +369,29 @@ In the above example we take three steps to compute the output value: we sample
 To make this more clear let us re-write the program as:
 
 ~~~~
-var A = flip();
-var B = flip();
-var C = [A, B];
-C;
+var a = flip()
```

Comment:
The text uses capitol letters (A,B,C) so let's stick with them in the code here?